server/api/contest_handler.py

```diff
@@ -10,8 +10,15 @@
     'contest_handler', __name__)
 
 
-@contest_handler.route('')
+@contest_handler.route('', methods=['GET'])
```

Comment:
i think this is okay to merge in for now, but you may have to change it again in a day or few hours!

https://github.com/hatchways/team-bubbletea/pull/66/files#diff-8402e43682ced11d9799215c5fa0234aR27

^ @john-rigas 's new helper function of authenticating users may help reduce your logic and increase security :)